platforms/android/library/src/main/java/io/element/android/wysiwyg/display/MentionDisplayHandler.kt

```diff
@@ -14,4 +14,9 @@ interface MentionDisplayHandler {
      * Return the method with which to display an at-room mention
      */
     fun resolveAtRoomMentionDisplay(): TextDisplay
+
+    /**
+     * Return true if the given URL is a mention one
+     */
+    fun isMention(url: String): Boolean
```

Comment:
@ganfra asked if we could make the RTE take care of detecting mentions instead of relying in the client's implementation. But you're probably right, the RTE library itself should be agnostic about this and don't rely on the Matrix mentions at all. 

Should we publish a separate library to handle this?

Reply:
Done in https://github.com/matrix-org/matrix-rich-text-editor/pull/854/commits/ce455e76045cef08db2c13ccc4e3486b315af47e